data/org.freedesktop.impl.portal.PermissionStore.xml

```diff
@@ -43,7 +43,7 @@
 
       This document describes version 2 of the permission store interface.
   -->
-  <interface name='org.freedesktop.impl.portal.PermissionStore'>
```

Comment:
The changes from `'` to `"` for consistency are a great cleanup. Can you split them into their own commit?